frontend/components/shared/footer/BookingFooter.tsx

```diff
@@ -0,0 +1,185 @@
+import { Link } from '@nextui-org/react'
+import Image from 'next/image'
+const COMPANY_CONTACT_METHODS = [
+  {
+    name: 'Customer Support',
+    link: '#',
+  },
+  {
+    name: 'Service Guarantee',
+    link: '#',
+  },
+  {
+    name: 'Website Feedback',
+    link: '#',
+  },
+]
+
+const COMPANY_INFOS = [
+  {
+    name: 'About Our Company',
+    link: '#',
+  },
+  {
+    name: 'Terms and Condition',
+    link: '#',
+  },
+  {
+    name: 'Privacy and Cookies',
+    link: '#',
+  },
+]
+
+const COMPANY_PARTNER_METHODS = [
+  {
+    name: 'Partner Portal',
+    link: '#',
+  },
+  {
+    name: 'Affiliate Program',
+    link: '#',
+  },
+]
+
+const PAYMENT_METHOD_LOGOS = [
+  {
+    text: 'Visa',
+    src: '/payment_logos/visa.png',
+  },
+  {
+    text: 'Mastercard',
+    src: '/payment_logos/mastercard.png',
+  },
+  {
+    text: 'G-Cash',
+    src: '/payment_logos/Gcash.png',
+  },
+  {
+    text: 'GrabPay',
+    src: '/payment_logos/GrabPay.png',
+  },
+  {
+    text: 'PayMaya',
+    src: '/payment_logos/PayMaya.png',
+  },
+  {
+    text: 'BPI',
+    src: '/payment_logos/BPI.png',
+  },
+  {
+    text: 'Security Bank',
+    src: '/payment_logos/SecurityBank.png',
+  },
+  {
+    text: 'AUB',
+    src: '/payment_logos/AUB.png',
+  },
+  {
+    text: 'BillEase',
+    src: '/payment_logos/BillEase.png',
+  },
+  {
+    text: 'CoinsPH',
+    src: '/payment_logos/CoinsPH.png',
+  },
+  {
+    text: '7-11',
+    src: '/payment_logos/7_eleven.png',
+  },
+  {
+    text: 'Cebuana Lhuiller',
+    src: '/payment_logos/CebuanaLL.png',
+  },
+  {
+    text: 'M Lhuiller',
+    src: '/payment_logos/M_LL.png',
+  },
+  {
+    text: 'Landbank',
+    src: '/payment_logos/Landbank.jpg',
+  },
+]
```

Comment:
These values are the same as in Footer.tsx, you can just export them from Footer.tsx and import them here in BookingFooter. This will ensure that the code is more maintainable